app/models/concerns/hyku_addons/generic_work_overrides.rb

```diff
@@ -124,7 +124,7 @@ module GenericWorkOverrides
       end
 
       self.json_fields += %i[editor current_he_institution]
-      self.date_fields += %i[event_date related_exhibition_date]
+      self.date_fields += %i[event_date related_exhibition_date date_created]
```

Comment:
I don't believe that `date_created` is a field on our models. (It might be in upstream GenericWork but not on ours.)

Reply:
You are correct. I added it as a test because `hyrax_work` is only giving me `date_created` in the json... and I forgot to remove it.